src/components/MonitorSettingDialog.vue

```diff
@@ -90,25 +93,7 @@ export default {
          * @returns {void}
          */
         toggleLink(groupIndex, index) {
-            this.$root.publicGroupList[groupIndex].monitorList[index].sendUrl = !this.$root.publicGroupList[groupIndex].monitorList[index].sendUrl;
-        },
-
-        /**
-         * Should a link to the monitor be shown?
-         * Attempts to guess if a link should be shown based upon if
-         * sendUrl is set and if the URL is default or not.
-         * @param {object} monitor Monitor to check
-         * @param {boolean} ignoreSendUrl Should the presence of the sendUrl
-         * property be ignored. This will only work in edit mode.
-         * @returns {boolean} Should the link be shown?
-         */
-        showLink(monitor, ignoreSendUrl = false) {
-            // We must check if there are any elements in monitorList to
-            // prevent undefined errors if it hasn't been loaded yet
-            if (this.$parent.editMode && ignoreSendUrl && Object.keys(this.$root.monitorList).length) {
-                return this.$root.monitorList[monitor.element.id].type === "http" || this.$root.monitorList[monitor.element.id].type === "keyword" || this.$root.monitorList[monitor.element.id].type === "json-query";
-            }
-            return monitor.element.sendUrl && monitor.element.url && monitor.element.url !== "https://" && !this.editMode;
+            this.$root.publicGroupList[groupIndex].monitorList[index].sendUrl = this.$root.publicGroupList[groupIndex].monitorList[index].sendUrl ? 0 : 1;
```

Comment:
Why not the code beforehand? changing this to be a bool->number ambiguates/complicates some semantics.

Reply:
@CommanderStorm 
Thank you! I have addressed in the following commit. 2fb3638bd70fc2ac34b2276eb2c47d59b9badc75

The reference used is this.
https://github.com/louislam/uptime-kuma/blob/46d8744fa40cf6a5773578252af655058437b1e6/server/model/status_page.js#L404-L405